mozilla/fx-private-relay
# change locales/ to a submodule

This changes the app repo to use the new l10n repo for its `locales/` files.

Test:
1. Check out this branch 
2. `git submodule update --init`
3. Go to http://127.0.0.1:8000/
   * All the UI strings should still work

Notes:

I've updated the `.circleci/config.yml` file so it will automatically get the submodule contents before it builds. Heroku automatically detects submodules and updates them when deploying code.

From now on, when we want to update the translations for the app - e.g., for a release - we will need to:

1. `git submodule update --remote`
2. `git add privaterelay/locales`
3. `git commit -m "update translations"`
4. `git push`
